index.js

```diff
@@ -0,0 +1,11 @@
+const burgerButton = document.getElementById('burger');
+const closeButton = document.getElementById('close');
+const mobileNav = document.querySelector('.mobile-nav');
+
+burgerButton.addEventListener('click', () => {
+  mobileNav.style.display = 'flex';
+});
+
+closeButton.addEventListener('click', () => {
```

Comment:
Please kindly add event listeners as well. When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears. This was one of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#project-requirements). Please see snapshot:

<img width="1010" alt="Screen Shot 2021-07-26 at 00 24 29" src="https://user-images.githubusercontent.com/60043698/126911016-4d29397d-43d1-4e0d-8965-4a52772de811.png">